### ISSUE TITLE
Could not resolve project :steppers

### ISSUE THREAD

**Issue by Rudloff** [TheAndroidMaster/Status#129]
Hello,

I'm trying to package your app for F-Droid (#116).

I see the app needs a steppers library which is not included in this repository.

I tried to clone [your fork of the library](https://github.com/TheAndroidMaster/Steppers) in `steppers/` but I still get the same error:
```
Could not determine the dependencies of task ':app:compileReleaseJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:releaseCompileClasspath'.
   > Could not resolve project :steppers.
     Required by:
         project :app
      > Unable to find a matching configuration of project :steppers: None of the consumable configurations have attributes.
```

Would it be possible to add some documentation on how to build this?

**Comment by TheAndroidMaster** [comment]
Sure. I don't believe that there is actually any reason to use my fork of the library over the dependency anymore as a new version has been released since I forked it, I just haven't changed it back yet.

The library wasn't included in the repository because I was unsure how the Apache license was applied to it (the library has part of it in its README but no file in the actual repository) and I'm scared of legal things. 

I'll let you know when I've fixed the dependency.
